<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HEME</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/3d_blood.jpg</normaloff>:/images/3d_blood.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
font: 10pt &quot;Segoe UI&quot;;
}
QLineEdit {
margin: 5px;
padding: 5px;
}
QPushButton {
border: 1px solid #000;
margin: 10px;
padding: 3px;
min-width: 100px;
min-height:30px;
background-color: #FFF;
}
QPushButton:hover {
background-color: #EEE
}
QLabel {
margin: 10px;
}
QCheckBox {
margin: 5px;
padding: 5px;
}
otherRadioButton {
margin: 5px;
padding: 5px;
max-width:70px;
}
QGroupBox {
margin: 5px;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topHorizontalLayout">
      <item>
       <widget class="QLabel" name="inputlabel">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Input Image:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputLineEdit">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="inputPushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="filterTypeGroupBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Filter type</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="medianBlurRadioButton">
         <property name="text">
          <string>Median Blur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="gaussianBlurRadioButton">
         <property name="text">
          <string>Gausian Blur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="otherRadioButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Other</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="otherComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Output Image:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputLineEdit">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outputPushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="displayImageCheckBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Display after saving</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
